spec/system/admin_sidebar_navigation_spec.rb

```diff
@@ -50,8 +50,8 @@
   it "collapses sections by default" do
     visit("/admin")
     links = page.all(".sidebar-section-link-content-text")
-    expect(links.count).to eq(3)
-    expect(links.map(&:text)).to eq(["Dashboard", "Users", "All Site Settings"])
+    expect(links.count).to eq(4)
+    expect(links.map(&:text)).to eq(["Dashboard", "Users", "All Site Settings", "What's New"])
```

Comment:
It seems a bit redundant to assert on the count and then the exact elements, since the latter covers the former and more. 🙂

Reply:
Yeah will follow this up in another PR